bridges/snowbridge/pallets/ethereum-client/tests/electra/execution-proof.json

```diff
@@ -0,0 +1,54 @@
+      "0xf72c17e4b0335c0ff944ca33455734793315afe7eec60294370ce4449e62ddeb",
+      "0xbefa57233af9add021a971ba8f13f24fed9bb9041360795e037b60b28b4fe93c",
+      "0xde5943a938b973bf2cf2ba7b05dc8ed6113e9d1659e2b40e0de0c91de5cd67da",
+      "0x401318565c107ddd5b2de4671a798589088c080c26bcbeb6701773c3f25f97d6",
+      "0xb3555c1cf5898f9d83580757a6d31e603f723b19f4b4bbcfeca701222e8b560e",
+      "0xab7c47c9f8e4b6439647bcd53e717fb1069e7fbce7ed548e724c0a8ef364d6d2",
+      "0xe787c98181293e74b6973fb75f92b2e323119c095485178c1516df69fbfac2cb",
+      "0xc2a7138ee8ce76654281b297707b67229e9334a1d4fdcc792c87691e4f18e2c5",
+      "0xd23f0d2ca62ee6840fdbce17673c4ef6a29266a07b3fe504d54be0ed6c30ed53",
+      "0x01462e461c6fe4865b751c1a55f1bf21cacb611dbe795b1562bc76c1c36baf64",
+      "0x6cf04127db05441cd833107a52be852868890e4317e6a02ab47683aa75964220",
+      "0xb7d05f875f140027ef5118a2247bbb84ce8f2f0f1123623085daf7960c329f5f"
+    ],
+    "finalized_block_root": "0xeadd60f47f74c01dc12c7dc892490f549e0664e18df3dd27b9a80dd7df2b1f23"
+  },
+  "execution_header": {
+    "Deneb": {
+      "parent_hash": "0xbddc8ab20335a9d9448ac21b56ada15b59974da43d25f5a299784f3b9cac31a4",
+      "fee_recipient": "0x0000000000000000000000000000000000000000",
+      "state_root": "0x466f0a0b0437f2f30d2417914ae88d0aec5b823ef99e6dec8000a65030e5ded1",
+      "receipts_root": "0xf87cce309dcadb263aa9dda92bf74e6218b5198b9cb315c98d9be72376c203ac",
+      "logs_bloom": "0x00000000000000000000000000000000000000000000004000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000080000000000000000000000000000004000000000080000000000000000000000000000000000010100000000000000000000000000000000000000000000000000000000000000001000000000080000000000000000000000040000000000000000002000002000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000100000000000000000200000000000010",
+      "prev_randao": "0xd4e5c2b7df821e8cf57b0fdc1fda4658ec898f87dbd8554e2379423daabfd953",
+      "block_number": 1471,
+      "gas_limit": 30000000,
+      "gas_used": 123189,
+      "timestamp": 1736963157,
+      "extra_data": "0xd883010e0c846765746888676f312e32332e33856c696e7578",
+      "base_fee_per_gas": 7,
+      "block_hash": "0x8501525d685416e05beb2d5dca1ad7702e1e55c9f1c715a6c451e4538c900ddf",
+      "transactions_root": "0xc3a1efbc96ee4b3700a0674ea64bcbc3e3818144a7a297a1f7abc51826186a1d",
+      "withdrawals_root": "0x792930bbd5baac43bcc798ee49aa8185ef76bb3b44ba62b91d86ae569e4bb535",
+      "blob_gas_used": 0,
+      "excess_blob_gas": 0
+    }
+  },
+  "execution_branch": [
+    "0xcc390acbae705724d483d45371bf243e5ca7698e8d333be9eb4f1e37dc65919a",
+    "0xb46f0c01805fe212e15907981b757e6c496b0cb06664224655613dcec82505bb",
+    "0x6dd3b9955d892d92338b19976fd07084bfe88a76c3063482b7f30ee60feb2a58",
+    "0x940522efb5e2581496c5e254e388423781dd70d9ce959cc5ff65382471de0b16"
+  ]
+}
```

Comment:
missing final newline

Reply:
Addressed in https://github.com/paritytech/polkadot-sdk/pull/7075/commits/a25a8b834054b8770f58c1037490c81b94770a9b.